src/track/track.h

```diff
@@ -371,13 +371,12 @@ class Track : public QObject {
     void slotBeatsUpdated();
 
   private:
-    // Set a unique identifier for the track. Only used by
-    // GlobalTrackCacheResolver!
+    /// Set a unique identifier for the track.
+    /// Only used by GlobalTrackCacheResolver!
     void initId(TrackId id);
-    // Reset the unique identifier after purged from library
-    // which undos a previous add. Only used by
-    // GlobalTrackCacheResolver!
-    void resetId();
+    /// Remove the TrackId upon purging from library.
+    /// Only used by GlobalTrackCacheResolver!
+    void unsetId();
```

Comment:
Renaming this function is not necessary. It doesn't improve the situation, "unset" is neither more concise nor less inconsistent than "reset".

Please avoid arbitrary renamings just for satisfying your personal habits and preferences. The term "unset" is not used anywhere else in the code base!

Reply:
I removed the function and then added it back again as I realized it is needed because of the mutex, that's how that happened.

I personally was confused a bit by the term reset - in my use of english it implies that it is setting something, i.e. the track gets a new id. But instead the id is removed completely. I was also considering `clear` instead.

But maybe this is just my English - I will readily adjust if you disagree.

Reply:
Please revert all the unnecessary changes.

Reply:
`reset` is standard naming, e. g. for smart pointers.